ansibullbot/triagers/ansible.py

```diff
@@ -1113,6 +1114,11 @@ def create_actions(self, iw, actions):
             if comment not in actions.comments:
                 actions.comments.append(comment)
 
+        # performance issue/PR
+        if self.meta['is_performance']:
+            if 'performance' not in iw.labels:
```

Comment:
Why not use one `if` conditional instead of double indenting body with two? Clause would perfectly fit in a single line.

Reply:
Mostly to keep style consistent with the rest of the code.